apps/api/env/.env.functional.test

```diff
@@ -38,3 +38,6 @@ AMPLITUDE_SAMPLING=1
 NOTIFICATIONS_API_BASE_URL=http://localhost:3081
 
 FEATURE_FLAGS_ENABLE_ALL=true
+
+PROXY_API_URL=https://api.sandbox-01.aksh.pw:443
+PROXY_RPC_URL=https://rpc.sandbox-01.aksh.pw:443
```

Comment:
non-blocking: i don't thing any client cares if this url is a proxy or not. it would still treat it as a API/RPC endpoint, right? perhaps renaming it might not make much sense.

Reply:
These are actually for the rpc proxy specifically